core/src/scene/sceneLoader.cpp

```diff
@@ -544,7 +544,7 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
                 std::lock_guard<std::mutex> lock(m_textureMutex);
 				auto texture = scene->getTexture(name);
                 if (texture) {
-                    if (!texture->loadImageFromMemory(ptr, dataSize, false)) {
+                    if (!texture->loadImageFromMemory(ptr, dataSize, true)) {
```

Comment:
Good call - definitely not threadsafe: https://github.com/nothings/stb/blob/master/stb_image.h#L931-L936

I can't think of a reason to load images in the opposite orientation, so I agree on getting rid of this parameter. I'll update.